packages/expo-updates/e2e/__tests__/utils/simulator.android.ts

```diff
@@ -0,0 +1,37 @@
+import spawnAsync from '@expo/spawn-async';
+import path from 'path';
+
+const ADB_PATH = (function () {
+  if (process.env.ADB_PATH) {
+    return process.env.ADB_PATH;
+  }
+  if (process.env.ANDROID_SDK_ROOT) {
+    return path.join(process.env.ANDROID_SDK_ROOT, 'platform-tools', 'adb');
+  }
```

Comment:
Add support for ANDROID_HOME as well: https://developer.android.com/studio/command-line/variables